fedmsg/consumers/ircbot.py

```diff
@@ -307,9 +313,10 @@ def prettify(self, topic, msg, pretty=False, terse=False, short=False):
                     link = fedmsg.meta.msg2link(msg, **self.hub.config)
 
                 if link and short:
-                    dagd = 'http://da.gd/s'
-                    resp = requests.get(dagd, params=dict(url=link))
-                    link = resp.text.strip()
+                    if callable(short):
```

Comment:
Perhaps it would be better to just let a user configure the service URL and require that the service return the shortened link as the response body to a GET with the ``url`` parameter. Will that meet your needs?